doc/whats_new.rst

```diff
@@ -301,6 +301,8 @@ Changelog
 
 - Handle different time vectors in topography plots using :func:`mne.viz.plot_evoked_topo` by `Jussi Nurminen`_
 
+- Speed up :func: `mixed_norm` if the :solver: parameter is set to `bcd` using :func:`scipy.linalg.get_blas_func`.
```

Comment:
There is a space between the `:func:` and `mixed_norm` so it will not render properly

Reply:
Also it's scipy.linalg.get_blas_funcs (plural)